examples/plotting/plot_4_partial_dependencies.py

```diff
@@ -99,16 +99,16 @@
 import time
 
 experiment = get_experiment("2-dim-exp", storage=storage)
-start = time.clock()
+start = time.process_time()
```

Comment:
I think `perf_counter` would be better than `process_time`. https://docs.python.org/3/library/time.html#time.perf_counter